1.8/tutorial/service-discovery.md

```diff
@@ -0,0 +1,65 @@
+---
+post_title: Connecting Apps/Service Discovery
+nav_title: Service Discovery
+menu_order: 4
+---
+
+# Prerequisites
+In order to get started with this tutorial, you should have access to a running DC/OS OSS cluster and the DC/OS CLI installed and configured.
+Furthermore, we expect you have just deployed app1 in the previous step of this tutorial.
+
+# Objective
+Our application in the previous part used`redis.marathon.l4lb.thisdcos.directory:6379` as the address for connecting to redis. As redis might be running on any agent in the cluster (and furthermore on different ports), how does this address link to the actual running redis instance?
+In this section, we will learn about DC/OS service discovery by exploring the different options for service discovery for apps in DC/OS.
+
+# Steps
+  [Service discovery](https://dcos.io/docs/1.8/usage/service-discovery/)  is very important as we actually want to be ignorant (at least most of the time) about where our apps are running, but still be able to connect to them. This is especially true in cases where apps fail and might be restarted on a different host.
+  DC/OS provides two options for service discover: Mesos-DNS and Named virtual IPs.
+  * Let us shh into our cluster and see how this works: `dcos node ssh --master-proxy --leader`,
+  * [Mesos-DNS](https://dcos.io/docs/1.8/usage/service-discovery/mesos-dns/) assigns a Mesos-DNS for every Marathon app.  The naming pattern is  *task.scheduler.mesos* and as the default scheduler for jobs is `marathon`, so the Mesos-DNS name for our redis service is *redis.marathon.mesos*.
+
+  Let us use [dig](https://en.wikipedia.org/wiki/Dig_(command)) to retrieve the *A*ddress record: `dig redis.marathon.mesos`.
+     The answer should contain something like the following response:
+
+  ~~~~
+  ;; ANSWER SECTION:
+  redis.marathon.mesos. 60  IN  A 10.0.0.43
+  ~~~~
+
+  The response tells us that the host for the 'redis.marathon.mesos'' is 10.0.0.43.
```

Comment:
remove "the" from the sentence above. And I think you have two `''` instead of one ``` after `mesos`.